app/BUILD.bazel

```diff
@@ -573,6 +578,14 @@ kt_android_library(
     ],
 )
 
+# Library for scope annotations required to build views and view_model libraries.
```

Comment:
Comment seems wrong--please update to describe what this library actually contains.

Reply:
My bad - done